log/context.go

```diff
@@ -88,8 +90,9 @@ func WithModule(ctx context.Context, module string) context.Context {
 // is set, an empty string is returned.
 func GetModulePath(ctx context.Context) string {
 	module := ctx.Value(moduleKey{})
+	// if no module is yet defined, then add swarmkit to the module path
 	if module == nil {
-		return ""
+		return "swarmkit"
```

Comment:
Should we append "swarmkit" even if the module is not nil ?

Reply:
I think the argument for doing this is to indicate that the error was first seen in swarmkit. But I can see that this might be confusing because that doesn't necessarily mean swarmkit failed or did something wrong. Probably better to add a module than not, though.

Reply:
Just leave it the way it is. No need for a double default.